hypertrace-graphql-entity-schema/src/main/java/org/hypertrace/graphql/entity/schema/argument/IncludeNonLiveEntitiesArgument.java

```diff
@@ -0,0 +1,7 @@
+package org.hypertrace.graphql.entity.schema.argument;
+
+import org.hypertrace.core.graphql.deserialization.PrimitiveArgument;
+
+public interface IncludeNonLiveEntitiesArgument extends PrimitiveArgument<Boolean> {
+  String ARGUMENT_NAME = "includeNonLiveEntities";
```

Comment:
I know we haven't come up with a better name, but having a negation inside  the boolean like this is just confusing. what about `includeInactive` (entity is already implied given that it's an arg to the entity api)?